internal/user/usercontroller/user_controller_test.go

```diff
@@ -334,17 +334,25 @@ func TestGetConnectedUsers_ReturnsConnectedUsers(t *testing.T) {
 	mockStream1.EXPECT().Send(&usercontroller.DlcMessage{DestName: modelUser1.Name}).Times(1)
 	mockStream2.EXPECT().Send(&usercontroller.DlcMessage{DestName: modelUser2.Name}).Times(1)
 
-	userInfo1 := &usercontroller.UserInfo{Name: modelUser1.Name}
-	userInfo2 := &usercontroller.UserInfo{Name: modelUser2.Name}
-
-	mockStream3.EXPECT().Send(userInfo1).Times(1)
-	mockStream3.EXPECT().Send(userInfo2).Times(1)
+	expectedUsers := make(map[string]usercontroller.UserInfo)
+	expectedUsers[modelUser1.Name] = usercontroller.UserInfo{Name: modelUser1.Name}
+	expectedUsers[modelUser2.Name] = usercontroller.UserInfo{Name: modelUser2.Name}
+
+	mockStream3.EXPECT().Send(gomock.Any()).DoAndReturn(
+		func(info *usercontroller.UserInfo) error {
```

Comment:
I'm a bit picky, but I feel this wouldn't catch an issue if the stream returned two times the same username.

Reply:
even with the user found is removed from the map after ?

Reply:
and call only 2 times ?

Reply:
Good point sorry I had missed the deletion, then I guess it works!